topics/admin/tutorials/deployment-platforms-options/slides.html

```diff
@@ -0,0 +1,280 @@
+---
+layout: tutorial_slides
+topic_name: admin
+tutorial_name: deployment-platforms-options
+logo: "GTN"
+---
+layout: true
+name: left-aligned
+class: left, middle
+
+---
+
+layout: true
+class: center, middle
+
+---
+
+# Platform Options and Requirements
+
+.footnote[\#usegalaxy / @galaxyproject]
+
+---
+
+# Where can Galaxy run?
+* Cloud (SaaS)
+  - [usegalaxy.org](https://usegalaxy.org)
+  - [Public Galaxy Servers](https://wiki.galaxyproject.org/PublicGalaxyServers)
+  - [Amazon EC2](https://wiki.galaxyproject.org/CloudMan)
+  - Semi-private cloud (e.g.: [Genomics Virtual Lab](https://www.genome.edu.au/), [Jetstream](http://jetstream-cloud.org/))
+* Private cloud (build your own Galaxy SaaS)
+* Cloud (IaaS)
+  - Any cloud
+* Scalable Local Server
+  - Dedicated or shared compute cluster(s)
+  - Cloud compute resources
+* Standalone Local Server
+
+---
+# Choosing where to run
+
+__Public Prebuilt SaaS (usegalaxy.org, public servers)__
+
+- Quickest to use today, but probably not why you're here...
+- Institutional/Protected data a concern
+- Not covered in this training
+
+__Private Prebuilt SaaS (EC2, GVL, Jetstream) or build your own__
+
+- Great choices for people needing access to compute for a fixed time analysis
+- Not as conducive to collaboration, publishing
+- Covered: Wednesday: Using public and private cloud compute resources
+
+__Build your own Galaxy SaaS__
+- Personalized Galaxy instances for beginner-to-intermediate users
+- Requires a lot of infrastructure building
+- Covered: Wednesday: Using public and private cloud compute resources
+
+---
+# Choosing where to run
+
+__Scalable Local Server__
+- Permanent Galaxy server
+- Flexible compute scalability
+- Full privacy control
+- Covered: Wednesday: Connecting Galaxy to a compute cluster
+
+__Standalone Local Server__
+- Permanent Galaxy server
+- Full privacy control
+- Should only consider this in cases of expected light usage
+- Get a beefy server
+- Covered: Today
+
+---
+# Software Requirements
+
+Required:
+  - Galaxy is written in Python and depends on **Python 2.7**
+  - All major distros in wide circulation have 2.7, *except* RHEL<sup>[1]</sup> 6
+    - See: Software Collections for [RHEL](https://access.redhat.com/solutions/472793), [CentOS](https://wiki.centos.org/AdditionalResources/Repositories/SCL), [Scientific Linux](http://linux.web.cern.ch/linux/scl/)
+
+Optional (but not really):
+  - PostgreSQL
+    - Covered: Today
+  - uWSGI (will come with Galaxy from 18.01 onwards)
+    - Covered: Tuesday
+  - Reverse proxy server (nginx, Apache)
+    - Covered: Today
```

Comment:
I'd removed the references to Today/Tuesday

Reply:
What about including links to the mentioned training?

Reply:
Sure (together with removing the days).